docs/LinksAndButtons.md

```diff
@@ -0,0 +1,97 @@
+
+### Unique screen reader characteristics/features
+
+* VoiceOver allows users to use the rotor and self-select what they are going to navigate by (headings, links, buttons)
+* NVDA and JAWS have built-in ways to navigate by heading, unvisited links, visited links, buttons
+
+### Mobile Navigation
+
+* One user navigated the site using an iPhone and demonstrated mobile accessibility. The user employed the VoiceOver rotor as they had previously done on the desktop, allowing them to navigate by headings, links and buttons. When using the rotor, the user swiped right to advanced to the next identified element. When the rotor was turned off, it appeared swiping would read the next segment of the page, much as the Ctrl + Option + Right Arrow command would do on the desktop.
+* iOS VoiceOver has the same verbal settings and feel as Desktop VoiceOver, lending it a familiarity to Apple desktop users. Recommend future testing include mobile VoiceOver testing when possible.
+
+### Unique observations from our testers
+
+* One user was looking for a link to jump to a form farther down the page. Could be an anchor ID to a containing div, if the form is farther down the page from a descriptive block of text.
+* If the call to action, form, etc. is on the current page, use a link with descriptive “further down this page” language
+* If the call to action, form, etc, is on another page, use a link with descriptive “another page” language
+
+## Research findings <sup id="heading2">[2]</sup>
+
+It was the consensus of our test users that call to action links can be styled as buttons. The underlying HTML is an `<a>` tag that must meet the success criteria outlined below, and visually it will look like a button. Sighted users rely on this visual information, so having the call to action presented in a strong, clear manner is important. Screen reader users rely more on semantic meaning. Calls to action need to offer clear language about what we are asking users to do. If calls to action have short text blocks, say 2-4 words, consider adding a more descriptive aria-label for assistive devices.
+
+### Example link with aria-label for semantic meaning
+
+```html
+  <a
+    href=“#”
+    aria-label=“Start your application for health coverage on the next page”
+  >
+    Start your application
+  </a>
+```
+
+### Benefits for assistive devices
+
+* Sighted users will interpret “Start your application” and have the heading, sub-heading, and content around the CTA to determine intent
+* Screen reader users will hear “Start your application for health coverage on the next page” read out, and also have context because of the aria-label
+
+## Three rules for link and button success <sup id="heading3">[3]</sup>
+
+1. Links go to to another page OR jump to an anchor ID in the current page
+    1. SUCCESS CRITERIA:
+    2. Respond to ENTER keypress
+    3. The browser URL changes
+    4. HTML5 history object changes
```

Comment:
4 and 5 are good general guidance, but React Router handles this correctly for us, so we won't have to do anything special. I wonder if that will confuse someone working on an app here.

Reply:
@jbalboni I'll reframe the items and call out React Router specifically as solving these. I put them in thinking of the vanilla JS and static site scenarios. Can see how this could add confusion when working with React. Good call.